pages/agent/v3/elastic_ci_aws.md.erb

```diff
@@ -167,17 +169,46 @@ The AWS ECR options are powered by an embedded version of the [ECR plugin](https
 
 ## Elastic Stack CI releases
 
-We recommend running the latest release, which is available at `https://s3.amazonaws.com/buildkite-aws-stack/aws-stack.yml`, or on the [releases page](https://github.com/buildkite/elastic-ci-stack-for-aws/releases).
+We recommend running the latest stable release of the CloudFormation template,
```

Comment:
If possible avoid using `we` as it isn't super clear who we is, and what their motivation is. (Buildkite? Docs team? Elastic stack team? )